mlflow/genai/judges/builtin.py

```diff
@@ -222,35 +222,47 @@ def is_correct(
     model: str | None = None,
 ) -> Feedback:
     """
-    LLM judge determines whether the given response is correct for the input request.
+    LLM judge determines whether the expected facts are supported by the response.
+
+    This judge evaluates if the facts specified in ``expected_facts`` or ``expected_response``
+    are contained in or supported by the model's response. It answers: "Does the response
```

Comment:
nit: the "It answers: ..." seems a bit redundant and doesn't mention expected response - wdyt of removing this?

Reply:
Agreed! Removed :)